except Exception as ex:

except ValueError as ve:

except:

except Exception: